app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php

```diff
@@ -33,6 +33,16 @@
  */
 class Mage_Currencysymbol_Block_Adminhtml_System_Currencysymbol extends Mage_Adminhtml_Block_Widget_Form
 {
+    /**
+     * @var string
+     */
+    private $_blockGroup;
```

Comment:
I think that _blockGroup, _controller should be protected or deleted. As I see in project this attributes is not used. From this class is not extended any class, so I think, that these variables are pointless.

Reply:
Fixed.